econml/sklearn_extensions/ensemble.py

```diff
@@ -756,3 +761,38 @@ def predict_interval(self, X, alpha=.1, normal=True):
         lower_pred = scipy.stats.norm.ppf(
             alpha / 2, loc=y_point_pred, scale=pred_stderr)
         return lower_pred, upper_pred
+
+    @property
+    def feature_importances_(self):
+        """
+        The impurity-based feature importances.
+
+        The higher, the more important the feature.
+        The importance of a feature is computed as the (normalized)
+        total reduction of the criterion brought by that feature.  It is also
+        known as the Gini importance.
+
+        Returns
+        -------
+        feature_importances_ : ndarray of shape (n_features,)
+            The values of this array sum to 1, unless all trees are single node
+            trees consisting of only the root node, in which case it will be an
+            array of zeros.
+        """
+        check_is_fitted(self)
+
+        def unnormalized_importances(tree):
+            return tree.tree_.compute_feature_importances(normalize=False)
+
+        all_importances = Parallel(n_jobs=self.n_jobs,
+                                   **_joblib_parallel_args(prefer='threads'))(
+            delayed(unnormalized_importances)(tree)
+            for tree in self.estimators_ if tree.tree_.node_count > 1)
+
+        if not all_importances:
+            return np.zeros(self.n_features_, dtype=np.float64)
```

Comment:
This isn't covered by any tests.

Reply:
I have added tests that call feature importances. Not sure why it isnt covered. Maybe you looked at the build before the last commit. This was added in the last commit

Reply:
Oh you mean the if not all_importances??

Reply:
I implemented the TODO which now makes the feature_importances_ more correct. So now this piece of code is actually erased. But it is replaced with more code in the parallel_add_trees function.